aspnetcore/security/authorization/limitingidentitybyscheme.md

```diff
@@ -145,3 +145,52 @@ In the preceding example, the "Over18" policy only runs against the identity cre
 [Authorize(Policy = "Over18")]
 public class RegistrationController : Controller
 ```
+
+## Use multiple authentication schemes
+
+Some apps may need to support multiple types of authentication. For example, your app might authenticate users from Azure Active Directory and from a users database. Another example is an app that authenticates users from both Active Directory Federation Services and Azure Active Directory B2C. In this case, the app should accept a JWT bearer token from several issuers.
+
+Add all authentication schemes you'd like to accept. For example, the following code in `Startup.ConfigureServices` adds two JWT bearer authentication schemes with different issuers:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    // Code omitted for brevity
+
+    services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
```

Comment:
@HaoK How should this code look for ASP.NET Core 1.x apps? This `JwtBearerDefaults.AuthenticationScheme` parameter is a 2.x feature.